code/modules/admin/verbs/mapping.dm

```diff
@@ -317,7 +317,7 @@ GLOBAL_VAR_INIT(say_disabled, FALSE)
 	//Also add the x
 	for(var/x_number in 1 to 4)
 		final.Insert(icon('icons/mob/screen_gen.dmi', "x[x_number == 1 ? "" : x_number]"), "x[x_number == 1 ? "" : x_number]")
-	fcopy(final, "icons/mob/landmarks.dmi")
+	fcopy(final, "icons/effects/landmarks_static.dmi")
```

Comment:
The code just above this is the code that is responsible for generating the job landmark icons. They are all automatically generated and if you want to change them, you need to update this to generate the icons that you want.

Reply:
well you know something new everyday, will fix that asap